samples/bluetooth/mesh/shell_client/README.rst

```diff
@@ -0,0 +1,193 @@
+
+Provisioning
+============
+
+The provisioning is handled by the :ref:`bt_mesh_dk_prov`.
+It supports four types of out-of-band (OOB) authentication methods, and uses the Hardware Information driver to generate a deterministic UUID to uniquely represent the device.
+
+Models
+======
+
+The following table shows the mesh shell client composition data for this sample:
+
+.. table::
+   :align: center
+
+   +-------------------------------+
+   | Element 1                     |
+   +===============================+
+   | Config Server                 |
+   +-------------------------------+
+   | Health Server                 |
+   +-------------------------------+
+   | Scheduler Client              |
+   +-------------------------------+
+   | Time Client                   |
+   +-------------------------------+
+   | Scene Client                  |
+   +-------------------------------+
+   | Light LC Client               |
+   +-------------------------------+
+
+The models are used for the following purposes:
+
+* The element contains a Config Server and a Health Server.
+  The Config Server allows configurator devices to configure the node remotely.
+  The Health Server provides ``attention`` callbacks that are used during provisioning to call your attention to the device.
+  These callbacks trigger blinking of the LEDs.
+* The remaining models are instances of the various clients described in the overview.
+  Lists of the available shell commands for the client models can be found on their :ref:`model pages <bt_mesh_models>`.
+
+The model handling is implemented in :file:`src/model_handler.c`, which uses the :ref:`dk_buttons_and_leds_readme` library to control the LEDs on the development kit.
+
+User interface
+**************
+
+Buttons:
+   Can be used to input the out-of-band (OOB) authentication value during provisioning.
+   All buttons have the same functionality during this procedure.
+
+LEDs:
+   Show the OOB authentication value during provisioning if the "Push button" OOB method is used.
+   First LED outputs the current light level of the Light Lightness Server in the first element.
+
+
+Configuration
+*************
+
+|config|
+
+Source file setup
+=================
+
+This sample is split into the following source files:
+
+* A :file:`main.c` file to handle initialization.
+* A file for handling mesh models, :file:`model_handler.c`.
+* A python script :file:`time_set.py` that contains a function that can be used to set the time status of a Time Server.
+* A python script :file:`toggle_onoff.py` that can be used to schedule periodic tasks to toggle a LED every 15s.
+* A python script :file:`toggle_scene.py` that can be used to schedule periodic tasks to recall a scene every 30s.
+* A python script :file:`store_scenes.py` that can be used to generate scenes to be used in the point above.
+
+
+FEM support
```

Comment:
This might not be relevant for the sample/should be tested before including

Reply:
To be honest I'm not quite sure. Just leave it as is for now.